dissect/target/helpers/loaderutil.py

```diff
@@ -1,12 +1,16 @@
+import logging
 import re
 import urllib
 from os import PathLike
 from pathlib import Path
 from typing import Optional, Tuple, Union
```

Comment:
While you're here, could you replace the type hint that currently uses `Tuple` with `tuple`? Thanks!